core/src/options/input_field.rs

```diff
@@ -149,7 +149,7 @@ impl ParseAttribute for InputField {
                 return Err(Error::duplicate_field_path(path).with_span(mi));
             }
 
-            self.with = Some(With::from_meta(mi)?);
+            self.with = Some(Callable::from_meta(mi)?);
```

Comment:
Nit: Other attributes are using `FromMeta::from_meta` instead of naming the target type. This should follow that pattern.